apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -839,7 +857,14 @@ def grade(element_html: str, data: pl.QuestionData) -> None:
         num_initial_correct, true_answer_length = grade_dag(
             submission, depends_graph, group_belonging
         )
-        if partial_credit_type is PartialCreditType.NONE:
+        if (
+            partial_credit_type is PartialCreditType.NONE
+            and grading_method is GradingMethodType.ORDERED
+        ):
+            student_answer = [ans["inner_html"] for ans in student_answer]
+            true_answer = [ans["inner_html"] for ans in true_answer_list]
+            final_score = 1 if student_answer == true_answer else 0
+        elif partial_credit_type is PartialCreditType.NONE:
```

Comment:
Why do you need this special case? 

won't the code just below this handle this case, just like it is handling the case of No partial credit for Ranking and DAG?

Reply:
Looks like you're right @SethPoulsen, the existing code for DAG and ranking handles no partial credit for ordered as well. I've went ahead and removed the unnecessary special case code.